src/dom/html/HtmlInputElementRe.re

```diff
@@ -0,0 +1,123 @@
+  [@bs.get] external placeholder : t_htmlInputElement => string = "";
+  [@bs.set] external setPlaceholder : (t_htmlInputElement, string) => unit = "placeholder";
+  [@bs.get] external readOnly : t_htmlInputElement => bool = "";
+  [@bs.set] external setReadOnly : (t_htmlInputElement, bool) => unit = "readOnly";
+  [@bs.get] external min : t_htmlInputElement => string = "";
+  [@bs.set] external setMin : (t_htmlInputElement, string) => unit = "min";
+  [@bs.get] external max : t_htmlInputElement => string = "";
+  [@bs.set] external setMax : (t_htmlInputElement, string) => unit = "max";
+  [@bs.get] external selectionStart : t_htmlInputElement => int = "";
+  [@bs.set] external setSelectionStart : (t_htmlInputElement, int) => unit = "selectionStart";
+  [@bs.get] external selectionEnd : t_htmlInputElement => int = "";
+  [@bs.set] external setSelectionEnd : (t_htmlInputElement, int) => unit = "selectionEnd";
+  [@bs.get] external selectionDirection : t_htmlInputElement => string = "";
+  [@bs.set] external setSelectionDirection : (t_htmlInputElement, string) => unit = "selectionDirection";
+
+  /* Properties not yet categorized */
+  [@bs.get] external defaultValue : t_htmlInputElement => string = "";
+  [@bs.set] external setDefaultValue : (t_htmlInputElement, string) => unit = "defaultValue";
+  [@bs.get] external dirName : t_htmlInputElement => string = "";
+  [@bs.set] external setDirName : (t_htmlInputElement, string) => unit = "dirName";
+  [@bs.get] external accessKey : t_htmlInputElement => string = "";
+  [@bs.set] external setAccessKey : (t_htmlInputElement, string) => unit = "accessKey";
+  [@bs.get] [@bs.return nullable] external list : t_htmlInputElement => option(Dom.htmlElement) = "";
+  [@bs.get] external multiple : t_htmlInputElement => bool = "";
+  [@bs.set] external setMultiple : (t_htmlInputElement, bool) => unit = "multiple";
+  /* TODO: files: FileList array. Returns the list of selected files. */
+  [@bs.get] external labels : t_htmlInputElement => array(Dom.nodeList) = "";
+  [@bs.get] external step : t_htmlInputElement => string = "";
+  [@bs.set] external setStep : (t_htmlInputElement, string) => unit = "step";
+  [@bs.get] [@bs.return nullable] external valueAsDate : t_htmlInputElement => option(Js.Date.t) = "";
+  [@bs.set] external setValueAsDate : (t_htmlInputElement, Js.Date.t) => unit = "valueAsDate";
+  [@bs.get] external valueAsNumber : t_htmlInputElement => float = "";
+
+  [@bs.send.pipe: t_htmlInputElement] external focus : unit = "";
+  [@bs.send.pipe: t_htmlInputElement] external blur : unit = "";
+  [@bs.send.pipe: t_htmlInputElement] external select : unit = "";
+  [@bs.send.pipe: t_htmlInputElement] external click : unit = "";
+  [@bs.send.pipe: t_htmlInputElement] external setSelectionRange : (int, int) => unit = "";
+  [@bs.send.pipe: t_htmlInputElement] external setSelectionRangeWithDirection : (int, int, string) => unit = "";
```

Comment:
This is missing an override of the name. The `string` argument here should ideally also be a variant.

Reply:
Fixed.